javascript/src/components/search/MessageFields.jsx

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+var React = require('react');
+var Immutable = require('immutable');
+var MessageFieldDescription = require('./MessageFieldDescription');
+
+var MessageFields = React.createClass({
```

Comment:
I'd add a PureRenderMixin here, even though it will probably only buy as little performance improvements